scripts/partition-graph.py

```diff
@@ -36,10 +39,10 @@
 DEFAULT_N_THREADS = 4
 
 
-def worker(queue, basename, stop_big_traversals):
+def worker(que, basename, stop_big_traversals):
```

Comment:
How about renaming this to something descriptive instead of this hamfisted truncation?